pandas/tests/indexes/multi/test_indexing.py

```diff
@@ -183,6 +183,155 @@ def test_get_indexer():
         idx1.get_indexer(idx2)
 
 
+def test_get_indexer_and_fill():
+    """ test getting an indexer for another index using the backfill method """
```

Comment:
comment instead of docstring, add a reference to the original GH sissue

Reply:
done, thanks for the tip